csrc/scheduler/reduction_heuristic.h

```diff
@@ -133,8 +135,15 @@ class ReductionParams : public HeuristicParams {
   // block_dim_inner_reduction_extra (usually TIDy)
   ParallelType block_dim_inner_reduction_extra = ParallelType::Serial;
 
-  // use shared memory for persistent buffer, if false, will use registers
+  // Use shared memory for persistent buffer, if false, will use registers.
+  // For innerOuterPersistentHeuristic, only the persistent tensors in the
```

Comment:
I wonder whether we should have these parameters as part of `ReductionParams`. It doesn't have the list of persistent buffers, but why should it have the list for the shared memory buffers?